.github/workflows/ci.yml

```diff
@@ -474,7 +474,7 @@ jobs:
 
       - name: Run Quarkus Tests in Docker
         run: |
-          ./mvnw clean install -nsu -B -f quarkus/tests/pom.xml -Dkc.quarkus.tests.dist=docker | misc/log/trimmer.sh
+          ./mvnw clean install -nsu -B -f quarkus/tests/pom.xml -Dkc.quarkus.tests.dist=docker -Dtest=StartCommandDistTest | misc/log/trimmer.sh
```

Comment:
Tests can also run from your IDE, as you know. That annotation is still useful to track which ones we don't run using a container so that we can either improve the support for containers.

One of the ideas behind the test suite is to make it possible to run any "dist" tests in whatever stack we should support. I hope we can remove it in the future once we have feature parity between the raw and container dist.

Here we are not really marking a test as not runnable on a container but focusing on reducing CI time.

> 
I'd prefer not to filter tests on the CI level. Maybe we can add some Maven profile (docker-smoke-tests?) as the single source of truth?

IMO, it is simpler to just ignore on GHA as this is the scope here.